bve/src/parse/train_dat/sections/version.rs

```diff
@@ -0,0 +1,38 @@
+use crate::parse::kvp::FromKVPValue;
+use bve_derive::FromKVPSection;
+
+#[derive(Debug, Default, Clone, PartialEq, FromKVPSection)]
+pub struct VersionSection {
+    #[kvp(bare)]
+    pub version: Version,
+}
+
+#[derive(Debug, Clone, PartialEq)]
+pub enum Version {
+    BVE120,
+    BVE121,
+    BVE122,
+    BVE2,
+    OpenBVE { version: String },
```

Comment:
BVEReborn when

Reply:
Hopefully never as I don't want to use this damn file format lmao